mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -434,7 +434,11 @@
     a section where users see a list of tabs that they have visited in the past few days -->
     <string name="customize_toggle_recently_visited">Recently visited</string>
     <!-- Title for the customize home screen section with Pocket. -->
-    <string name="customize_toggle_pocket">Pocket</string>
+    <string moz:RemovedIn="108" name="customize_toggle_pocket" tools:ignore="UnusedResources">Pocket</string>
+    <!-- Title for the customize home screen section with Pocket. -->
+    <string name="customize_toggle_pocket_2">Thought-provoking stories</string>
+    <!-- Summary for the customize home screen section with Pocket. -->
+    <string name="customize_toggle_pocket_summary">Articles powered by Pocket</string>
```

Comment:
@Alexandru2909: I noticed in other projects "Pocket" is made untranslatable, since it is a brand and shouldn't be touched. Could you update to word "Pocket" here to a placeholder instead, and add a comment saying the placeholder will be replaced by the "Pocket" brand name at run time?
I've also seen a couple instances in strings were "Pocket" is called out - probably good to just fix all instances with a placeholder, just in case. thanks!

Reply:
Sorry for the late reply, I must have missed this tag. Opened https://github.com/mozilla-mobile/fenix/issues/27691 in this scope.